### DISCUSSION TITLE
Testimonials

### DISCUSSION THREAD

**Discussion by tabuna** [orchidsoftware/platform#1306] [Show and tell]
There are many ways to contribute to the community without the desire or the required skills. One of the best is to attract new developers. If you know or meet people with programming skills, introduce them to the community and ask if they might be interested in developing Laravel Orchid.

But it would also be great to have a reviews section. I hope that over the years, the project has helped you. And I ask you to write about this so that your review would be shown on the main page.

<details><summary>Example</summary>
<img src="https://user-images.githubusercontent.com/5102591/96202540-2f544780-0f68-11eb-855f-6c88544dc536.png">
</details>

Please use the following template:

- Full name
- Country, city
- Photo
- Informative text of any length

**Comment by PSalador** [comment]
Павлов Павел
Россия. Шира
Для меня Orchid появился одновременно с Laravel, так как для обучения Laravel начал искать админку и выбор остановил на Orchid Platform. В данный момент все проекты  делаю на Orchid. Самое главное преимущество  Orchid, что можно быстро стартануть, и за очень короткое время можно перейти к самой сути проекта. А дальше легко масштабировать проект. Ещё огромный плюс в Orchid Platform, эта структура  организации кода изучение которой  помогает ускорить разработку и в других проектах.

**Comment by vladzorgan** [comment]
Владислав Пономарев
Россия, Краснодар

До встречи с Orchid постоянно приходилось писать админку с нуля. Удовольствие это было такое себе. Впервые услышал о Orchid в подкасте на "Пятиминутка PHP". Почитав документацию, решил попробовать и пользуюсь до сих пор. Orchid подходит под большинство, разрабатываемых мной проектов, и отлично масштабируется. Если у вас есть какие-то вопросы, то вам всегда помогут в официальном Telegram-чате проекта. Хотя ответы на большинство вопросов есть в документации.

**Comment by lemyskaman** [comment]
Lemys Alejandro López López
Venezuela, Valencia.

![IMG_20200430_222719](https://user-images.githubusercontent.com/4540813/98540669-af6a8600-2264-11eb-8bb4-ba75b0cfc3d7.jpg)



The laravel ecosystem is vast as mature. But as an aged web developer, I realize that "richness" is not enough,  I need a stable and well-designed open source platform, in terms of architecture and principles which allows to me and my team ship solutions fast, with an easy to maintain base code and without all the common fancy "automatic magic"  that might be complex to adapt and useless most of the time. Orchid-platform is all that and more, Orchid-Platform boost out our productivity in levels that i can't almost not to believe with beautiful and expressive resulting code. Orchid-platform is application-nature agnostic but it fits like no other package on administrative kind of Solutions.

**Comment by romansidorov** [comment]
Roman Sidorov
Russia, Moscow

A great solution. The key feature is a compromise between user-friendly interface and developer-friendly architecture. Ideal for administering small to medium sized projects. I recommend to consider it if you are a developer.

**Reply by lemyskaman** [reply]
Few words, but fully meaning full ones. Thats all i liked your testimonial a lot

**Comment by alash3al** [comment]
Mohamed Al Ashaal,
Egypt, Cairo

I liked Orchid as it gives me the power of control without the need of missy plugins/modules, I'm using it in many side-projects when comparing it to its similar solutions, I found that Orchid has a better architecture.

**Comment by goddas** [comment]
Max Minin,
USA, Florida.

I've used many Laravel admin panel/dashboard packages in the past, but none offer the simplicity and power of Orchid. It is really easy to get started and begin customizing the pages to your liking. The presence of a large community and good documentation is worth pointing out as well.
